src/shared/Core/DateTimeOffsetExtensions.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+
+namespace GitCredentialManager;
+
+public static class DateTimeOffsetExtensions
+{
+    private static readonly DateTimeOffset UnixEpoch =
+        new DateTimeOffset(1970, 1, 1, 0, 0, 0, TimeSpan.Zero);
```

Comment:
I was about to comment that `DateTimeOffset.UnixEpoch` already exists, but of course it does not in .NET Framework... 😢